contracts/TransferrableToken.sol

```diff
@@ -12,6 +12,7 @@ import "../node_modules/openzeppelin-solidity/contracts/introspection/ERC165.sol
 
 
 contract TransferrableToken is ERC165 {
+  function transfer(address _to, uint256 _valueOrTokenId) public;
```

Comment:
sorry, I messed up things here :( There is no `transfer` function in ERC721 interface, that's why we used `transferFrom` for everything.

So probably better remove `transfer` from `TransferrableToken` and either:
1. use approve + transferFrom as it was before
2. cast to ERC20 here https://github.com/leapdao/leap-contracts/pull/152/commits/ab2234c2258a435b5508c2e8d7a330bfefa2c005#diff-e8b2bda9737fa1a1ff063bbb22fb4cd5R202 so `transfer` can be used